README.md

```diff
@@ -197,9 +213,40 @@ output instead:
 ,,,1554210895
 ```
 
-It is possible to control the number of records to be put in a
-particular output file by setting `file.max.records`. By default, it is
-`0`, which is interpreted as "unlimited".
+**NB!**
+
+ - The `key.converter` property must be set to `org.apache.kafka.connect.converters.ByteArrayConverter`
+or `org.apache.kafka.connect.storage.StringConverter` for this data format.
+
+ - The `value.converter` property must be set to `org.apache.kafka.connect.converters.ByteArrayConverter` for this data format.
```

Comment:
Just checking: it's ok now for the key to be converted by `StringConverter`?

Reply:
For csv not sure. But this is a copy-paste from configuration example.

Reply:
Checked: yes. It is used in "groupByKey" use-case (there is a test with a same name)